yarn.lock

```diff
@@ -1896,7 +1896,7 @@ execa@^2.0.3, execa@^2.0.4:
     signal-exit "^3.0.2"
     strip-final-newline "^2.0.0"
 
-execa@^4.0.0, execa@^4.0.1, execa@^4.0.2:
+execa@^4.0.0, execa@^4.0.1:
```

Comment:
TBH I find it pretty unfortunate that these existing execa@^4.0.x entries are not combined with the new execa@^4.0.3 entry.